androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/ui/common/RtlModalDrawer.kt

```diff
@@ -0,0 +1,52 @@
+package co.nimblehq.avishek.phong.kmmic.android.ui.common
+
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.platform.LocalLayoutDirection
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.LayoutDirection
+
+/**
+ * ModalDrawer from right/end side
+ * https://issuetracker.google.com/issues/174514369#comment18
+ */
+@Composable
+fun RtlModalDrawer(
+    drawerContent: @Composable ColumnScope.() -> Unit,
+    modifier: Modifier = Modifier,
+    drawerState: DrawerState = rememberDrawerState(DrawerValue.Closed),
+    gesturesEnabled: Boolean = true,
+    drawerShape: Shape = MaterialTheme.shapes.large,
+    drawerElevation: Dp = DrawerDefaults.Elevation,
+    drawerBackgroundColor: Color = MaterialTheme.colors.surface,
+    drawerContentColor: Color = contentColorFor(drawerBackgroundColor),
+    scrimColor: Color = DrawerDefaults.scrimColor,
+    content: @Composable () -> Unit
```

Comment:
Please move `content` above `modifier`

Reply:
I'd like to move `content` to the bottom to follow the [trailing lambda convention](https://developer.android.com/jetpack/compose/kotlin#trailing-lambdas).